src/core/Makefile.am

```diff
@@ -199,6 +200,7 @@ noinst_HEADERS                      = \
     thread/thread_uris.hpp            \
     thread/topology.hpp               \
     utils/slaac_address.hpp           \
+    utils/jam_detector.hpp            \
```

Comment:
I added a build feature in the new commit push. Also added the files in VS project.